Doc/using/cmdline.rst

```diff
@@ -429,6 +429,8 @@ Miscellaneous options
      not be more verbose than the default if the code is correct: new warnings
      are only emitted when an issue is detected. Effect of the developer mode:
 
+     * :meth:`str.encode` and :meth:`bytes.decode` check their *errors*
```

Comment:
And `bytearray.decode()`, and `str()`, `bytes()` and `bytearray()` constructors...

Perhaps it may be better to say just "string encoding and decoding operations".

Reply:
Good idea: done. I took your sentence, but add the 3 most common functions as examples.